docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -147,19 +147,19 @@ Use the latest version of Prometheus or at least version 2.47.
        send_native_histograms: true
    ```
 
-## Scrape and send native histograms with Grafana Agent
+## Scrape and send native histograms with Grafana Alloy
 
-Use the latest version of the Grafana Agent in [Flow mode](/docs/agent/latest/flow/) (static mode support is coming soon).
+Use the latest version of [Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>).
 
-1. To enable scraping native histograms you need to enable the argument `enable_protobuf_negotiation` in the `prometheus.scrape` component:
+1. To scrape native histograms you need to enable the argument `enable_protobuf_negotiation` in the `prometheus.scrape` component:
```

Comment:
The `enable_protobuf_negotiation` argument is [deprecated in Alloy][1]. The latest docs [say](https://grafana.com/docs/alloy/latest/reference/components/prometheus.scrape/#scraping-behavior):

> To enable scraping of Prometheus’ native histograms over gRPC, the `scrape_protocols` should specify `PrometheusProto` as the first protocol to negotiate

[1]: https://grafana.com/docs/alloy/latest/reference/components/prometheus.scrape/#arguments